ota-plus-web/docs/modules/ROOT/pages/ota-workflow-campaign-manager.adoc

```diff
@@ -1,35 +1,33 @@
 = The Workflow of an OTA Update
 
-To deploy OTA updates in production, we use the concept of an update "campaign". We'll go into more details about campaigns later, but to understand the workflow, you first need to understand what goes into a campaign.
+To deploy OTA updates in production, we use the concept of a campaign. We'll go into more details about campaigns later, but to understand the workflow, you first need to understand what goes into a campaign.
 
 A campaign consists of the following elements:
 
-* The relevant software and the hardware that it supports.
-* The new software versions that you want to update to.
-* The groups of devices that you want target for this update.
+* The relevant software update and the supported hardware.
+* The new software versions to which you want to update.
+* The groups of devices that you want to target for this update.
+
+image::img::campaign-elements.png[image,600, align="left"]
 
-[cols="1a"]
-|===
-|image::img::campaign-elements.png[image,600,alight="center"]
-|===
 In the OTA workflow, you set up each of these elements in a specific order.
 
-.Overview of the OTA Update Worklow
+.Overview of the OTA update worklow
 [cols="3a,3a,3a,3a"]
 |===
-|1) Provision Devices|2) Group Devices|3) Upload software|4) Create update configuration
+|1) Provision devices|2) Upload a software version|3) Group devices|4) Create a software update
 
 |image::img::workflow-provision.png[image,100]
 Connect your devices to the OTA Connect server. The server stores the following information about each device:
 
-* The device identity,
+* The device identity
 * The hardware on the device
```

Comment:
```suggestion
```

Reply:
Ok, github's suggestion feature seems to be a bit buggy here and lost the other suggestions near this line. Here's what the list should be:

* The device name or identifier (often a VIN)
* The serial number and control unit type of each control unit in the device
* The software version that each control unit is currently running

Reply:
Changed